pdcupdater/handlers/retirement.py

```diff
@@ -0,0 +1,67 @@
+import logging
```

Comment:
I notice that this project doesn't seem to have an overall copyright file. You might want to add a copyright header on the two files in this PR.